config/sidekiq.yml

```diff
@@ -3,8 +3,10 @@
 :concurrency:  2
 :logfile: ./log/sidekiq.json.log
 :queues:
-  - content_store_high
-  - content_store_low
+  - content_store_high # Deprecated, to be removed once jobs completed
+  - content_store_low # Deprecated, to be removed once jobs completed
```

Comment:
Not all the uses of the PresentedContentStoreWorker have been removed, so I would hold off in marking the queues as deprecated.